src/PSR7/Validators/SerializedParameter.php

```diff
@@ -71,7 +72,7 @@ public static function fromSpec(CebeParameter $parameter): self
             }
 
             Validator::length(1, 1)->assert($content);
-        } catch (ExceptionInterface $e) {
+        } catch (Exception | ExceptionInterface $e) {
```

Comment:
[Union types](https://wiki.php.net/rfc/union_types_v2) were added in PHP 8, so I think this change would make this class and any others with a similar change in this PR unusable in older versions of PHP.

Reply:
This is not union types. Since PHP 7.1 you can specify multiple exceptions with pipe separator. https://www.php.net/manual/en/language.exceptions.php

Reply:
Ah! I stand corrected, and sorry for the noise. I learned something today and appreciate the explanation. :smile: